stubs/seaborn/@tests/stubtest_allowlist.txt

```diff
@@ -1,3 +1,5 @@
 seaborn._core.scales.(Pipeline|TransFuncs)  # aliases defined in `if TYPE_CHECKING` block
 seaborn.external.docscrape.ClassDoc.__init__  # stubtest doesn't like ABC class as default value
 seaborn.external.docscrape.NumpyDocString.__str__  # weird signature
+
+seaborn(\.regression)?\.lmplot  # the `data` argument is required but it defaults to `None` at runtime
```

Comment:
It is a shame we have to allow list the whole function and can't only list the offending parameter.

Reply:
X-ref https://github.com/python/mypy/issues/13703